webviz_subsurface/_abbreviations/volume_units.json

```diff
@@ -0,0 +1,26 @@
+{
+  "BULK_OIL": "m³",
+  "BULK_GAS": "m³",
+  "BULK_TOTAL": "m³",
+  "NET_OIL": "m³",
+  "NET_GAS": "m³",
+  "NET_TOTAL": "m³",
+  "PORV_OIL": "m³",
+  "PORV_GAS": "m³",
+  "PORV_TOTAL": "m³",
+  "PORE_OIL": "m³",
+  "PORE_GAS": "m³",
+  "PORE_TOTAL": "m³",
+  "HCPV_OIL": "m³",
+  "HCPV_GAS": "m³",
+  "HCPV_TOTAL": "m³",
+  "STOIIP_OIL": "Sm³",
+  "STOIIP_GAS": "Sm³",
+  "STOIIP_TOTAL": "Sm³",
+  "GIIP_OIL": "Sm³",
+  "GIIP_GAS": "Sm³",
+  "GIIP_TOTAL": "Sm³",
+  "RECOVERABLE_OIL": "Sm³",
+  "RECOVERABLE_GAS": "Sm³",
+  "RECOVERABLE_TOTAL": "Sm³"
+}
```

Comment:
Is it too risky to assume units? Or should e.g. user be able to overwrite this file by linking to another file? (if so, plugin by plugin, or for the whole app?). These units are also just my "best guess", would appreciate if e.g. @HansKallekleiv could take a look and see if they are correct. All units are volumes as far as I can understand, so is a question of standard conditions or not.

Reply:
For data coming from Eclipse vectors, we can utilize https://github.com/equinor/fmu-ensemble/pull/89 when that is merged. 

For data coming from (RMS) `.csv` export, I would hope/guess it is possible to also export units at the same time as the data is exported? @HansKallekleiv?

Reply:
This is basically a "sister file" to `volume_terminology.json`, so this applies mainly to the csv-files. We could also merge them, but by keeping them separate for now, I avoided having to rewrite code dependent on volume_terminology.json before we decide to go for this solution. Would of course be preferable to have units for all csv-file data as metadata from the original source, as we could then include verification of equal units when comparing data from different sources (e.g. grid data vs simulation data). The standard export files as of today are at least without specified units as far as I know.

Reply:
In Norway volumes are reported as mill Sm3 (oil) and bill Sm3 (gas). That might not be the case in other parts of the world (e.g. bbl are often used overseas). It is possible to export units from RMS, but there is no standard for this now.

I'd suggest we provide the option for the user  to overwrite keys from this file globally.

Reply:
The SI-prefix of these values are handled separately, these are only the units themselves. That is why I've written e.g. PORE_OIL in m^3, as Sm^3 doesn't make much sense for a pore volume to me. Currently dash has a bug when locking the SI prefix, but we could also include for the future a description of the prefix. E.g. that we have a single file on a form like (where prefix is optional):
{
    "RECOVERABLE_OIL": {"unit":  "Sm^3", "prefix": "M"},
    etc.
}

Another thing is that in different unit systems (like e.g. the American aka "field units" in the oil industry), the prefixes doesn't follow SI convention anyways (gas volumes in tcf -> trillion cubic feet). Allowing non-SI units will therefore open up for a whole new range of issues.